src/apis/general/enter/index.ts

```diff
@@ -0,0 +1,35 @@
+import { AuthToken } from "../../@types";
+
+interface EnterCredentials {
+  reservation_id: string;
+  guest_id: string;
+}
+
+export interface Methods {
+  /**
+   * 文化祭入場処理
+   *
+   * @remarks
+   * 必要な権限:
+   * - general
+   *
+   * @throws Error
+   * 400: Status Code とエラー内容が下記の通り対応
+   * | Status Code | Explanation |
+   * | :-- | :-- |
```

Comment:
```suggestion
   * | Status Code | Explanation |
   * | :---------- | :---------- |
```
swaggerからtsに変えるだったら変換後のでみる機会も少ないだろうし、この辺りスペース入れたりして列を揃えたいなとちょっと思います

Reply:
失礼致しました